examples/z_info.c

```diff
@@ -16,7 +16,7 @@
 #include "zenoh.h"
 
 void print_zid(const z_id_t *id, void *ctx) {
-    for (int i = 15; i >= 0; i--) {
+    for (int i = 0; i < 15; i++) {
```

Comment:
This needs to be `< 16` or `<=15`, but thanks for spotting the ordering issue :)